### ISSUE TITLE
[BUG] MS Word "Recents" tab do not open a previous open document

### ISSUE THREAD

**Issue by javiergonzper** [owncloud/ios#840]
## If you found a bug:
Recent documents open on the Document Provider at Microsoft Word can not be open from the "Recents" tab on MS Word.
### Steps to reproduce
1. Open a document on MS Word on iOS
2. Save the document after made some change
3. Try to open the same document again from the "Recents" tab on MS Word

![57ebd8b0-ea38-11e6-93da-7e6939a6b3af](https://cloud.githubusercontent.com/assets/1815050/22826541/54ba1e8a-ef93-11e6-99cd-9e878d19d526.gif)

### Expected behaviour
The document should be open

### Client
3.5.2

**ownCloud app version:**

**Device model:**
iPhone 6 plus

**More Info**

I checked the issue and the behavior of the Document Provider is:
1- Download the original file
2- Duplicate the original file as a temporal file on a Sandbox where Word can work
3- Edit the file with Word
4- Overwrite the original file with the temporal one
5- Add the new file to the "Upload queue".
6- Delete the temporal file

The problem happens when Word try to access to the temporal file again. But we have to delete it because if we do not do that the user could access to an old document because maybe the user modify in other computer or device.
When word try to access to the file again we just have an URL like this one:
file:///private/var/mobile/Containers/Shared/AppGroup/1B9F43A2-3A3B-466C-8F00-62F6E17EA059/File%20Provider%20Storage/file_7fa6e92ad461f8139283dc866174cf0b/Para%2520empezar%2520ahora%2520mismo.docx

With that information we can not re-download again the file to have it updated. Because there is not any reference to it.
We can continue working on this issue but you have to know that it is not trivial and probably we have to change how our Document Provider store the documents in order to have any reference to the old file to re-download it again.

**Comment by CorneelDragon** [comment]
MS Office also provides cloud storage providers better integration opportunities, but I don't know if (branded versions of) ownCloud are able to comply with their requirements: https://dev.office.com/programs/officecloudstorage

**Comment by michaelstingl** [comment]
Service providers like SURF or Sciebo could apply for integration, or ownCloud would need to run a central infrastructure where distributed individual ownCloud instances could connect to become integrated as cloud storage provider.

**Comment by javiergonzper** [comment]
Yes @CorneelDragon ownCloud and branded versions can be used as document providers of MS Word. This is a concrete issue.

**Comment by CorneelDragon** [comment]
Hi, any updates or a timeline yet?

**Comment by michaelstingl** [comment]
@CorneelDragon not yet. "Becoming [MS Office cloud storage provider](https://dev.office.com/programs/officecloudstorage)" is nothing that can be solved in the iOS repo.

**Comment by michaelstingl** [comment]
@CorneelDragon In iOS11, Word uses also the Files app integration. This looks more promising, and we are working on the best Files app integration possible.

please check my comment here:
https://github.com/owncloud/ios/issues/1038#issuecomment-374165166
